src/Site_Command.php

```diff
@@ -395,53 +402,40 @@ public function info( $args ) {
 			$this->populate_site_info( $args );
 		}
 		$ssl = $this->le ? 'Enabled' : 'Not Enabled';
-		EE::log( "Details for site $this->site_name:" );
+
 		$prefix = ( $this->le ) ? 'https://' : 'http://';
 		$info   = array(
-			array( 'Site', $prefix . $this->site_name ),
-			array( 'Access phpMyAdmin', $prefix . $this->site_name . '/ee-admin/pma/' ),
-			array( 'Access mailhog', $prefix . $this->site_name . '/ee-admin/mailhog/' ),
-			array( 'Site Title', $this->site_title ),
-			array( 'DB Root Password', $this->db_root_pass ),
-			array( 'DB Name', $this->db_name ),
-			array( 'DB User', $this->db_user ),
-			array( 'DB Password', $this->db_pass ),
-			array( 'E-Mail', $this->site_email ),
-			array( 'Cache Type', $this->cache_type ),
-			array( 'SSL', $ssl ),
+			array( 'Site', $prefix . $this->site_name . '/' ),
+			array( 'Admin Tools', $prefix . $this->site_name . '/ee-admin/' ),
 		);
 
 		if ( ! empty( $this->site_user ) && ! $this->skip_install ) {
 			$info[] = array( 'WordPress Username', $this->site_user );
 			$info[] = array( 'WordPress Password', $this->site_pass );
 		}
 
+		$info[] = array( 'DB Root Password', $this->db_root_pass );
+		$info[] = array( 'DB Name', $this->db_name );
+		$info[] = array( 'DB User', $this->db_user );
+		$info[] = array( 'DB Password', $this->db_pass );
+		$info[] = array( 'E-Mail', $this->site_email );
+		$info[] = array( 'Cache Type', $this->cache_type );
+		$info[] = array( 'SSL', $ssl );
+
 		\EE\Utils\format_table( $info );
 
 		\EE\Utils\delem_log( 'site info end' );
 	}
 
 	/**
 	 * Starts containers associated with site.
-	 * When no service(--nginx etc.) is specified, all containers will be restarted.
+	 * When no service(--mailhog etc.) is specified, all containers will be restarted.
 	 *
-	 * <site-name>
+	 * [<site-name>]
 	 * : Name of the site.
 	 *
 	 * [--all]
-	 * : Start all containers of site.
-	 *
-	 * [--nginx]
-	 * : Start nginx container of site.
-	 *
-	 * [--php]
-	 * : Start php container of site.
-	 *
-	 * [--mysql]
-	 * : Start mysql container of site.
-	 *
-	 * [--redis]
-	 * : Start redis container of site.
+	 * : Start all admin containers of site.
```

Comment:
What is the reason behind removing nginx, php, redis and mysql from the list?

How will someone start/stop let's say mysql container 🤔

Reply:
Got it. It can be done by site enable/disable.